pkg/cli/login/loginoptions.go

```diff
@@ -162,16 +162,26 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 	clientConfig.Host = o.Server
 	clientConfig.Insecure = o.InsecureTLS
 
-	if !o.InsecureTLS {
-		// use specified CA or find existing CA
+	if !clientConfig.Insecure {
```

Comment:
nit: Redundant change?

Reply:
Sounds good to me. My comment is already prefixed with `nit`. So this is not a blocker.